engine/gamesys/src/gamesys/resources/res_font.cpp

```diff
@@ -111,9 +115,13 @@ namespace dmGameSystem
 
     static dmRender::FontGlyph* GetGlyph(FontResource* resource, uint32_t codepoint)
     {
-        DynamicGlyph** dynglyphp = resource->m_DynamicGlyphs.Get(codepoint);
-        if (dynglyphp)
-            return &(*dynglyphp)->m_Glyph;
+        if (!resource->m_DynamicGlyphs.Empty())
+        {
+            DynamicGlyph** dynglyphp = resource->m_DynamicGlyphs.Get(codepoint);
+            if (dynglyphp)
+                return &(*dynglyphp)->m_Glyph;
+            return 0;
+        }
```

Comment:
New change. Don't fallback to any potential offline generated glyphs. Those are only present because our pipeline didn't support an empty set of glyphs (yet).